api/v1beta1/machinedeployment_types.go

```diff
@@ -97,6 +99,12 @@ type MachineDeploymentSpec struct {
 	// +optional
 	Replicas *int32 `json:"replicas,omitempty"`
 
+	// RolloutAfter is a field to indicate a rollout should be performed
```

Comment:
nit: can we add include an example of a valid input in the comment?

Reply:
+1

Reply:
While we're on it. Let's add the same example to the rolloutAfter field in KCP